samcli/commands/check/lib/command_context.py

```diff
@@ -68,7 +81,7 @@ def transform_template(self):
 
         return converted_template
 
-    def read_sam_file(self):
+    def read_sam_file(self) -> Any:
```

Comment:
should the return type be `Dict` instead of `Any`? (assuming `yaml_parse` is returning a dict object)

Reply:
I'll look into it again, but for some reason pylint thinks that it returns an `Any`. Putting anything else there creates a pylint error. For most things that return `Any`, I get this problem.